apps/src/sites/studio/pages/teacher_dashboard/show.js

```diff
@@ -130,23 +131,27 @@ $(document).ready(function () {
   };
 
   const getV2TeacherDashboard = () => {
-    const selectedSectionFromList = window.location.pathname.includes(
-      '/teacher_dashboard/home'
-    )
-      ? sections[0]
-      : sections.find(s => s.id === section.id);
-    const selectedSection = {...selectedSectionFromList, ...section};
+    if (sections.length > 0) {
```

Comment:
nit. Could we instead have `if (sections.length === 0) return <TeacherHomepage />;`? I think that would make it a bit clearer what the logic is since it puts the special case in the if statement at the top